src/gluonts/time_feature/seasonality.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+import logging
+
+import pandas as pd
+
+
+logger = logging.getLogger(__name__)
+
+
+DEFAULT_SEASONALITIES = {"H": 24, "D": 1, "W": 1, "M": 12, "B": 5}
```

Comment:
I believe we should change "D": 7, however, that is for another PR.

Reply:
It's strange, agreed. Also `W` could be `52`.

Are these seasonalities defined by M4, or something?